blackbox/requirements.txt

```diff
@@ -4,7 +4,7 @@ Babel==2.3.4
 colorama==0.3.7
 crash==0.20.0
 crate==0.19.0
-crate-docs-theme==0.5.18
+crate-docs-theme==0.5.21
```

Comment:
I think we should remove the version completely so it pulls the latest and have the same behaviour as on RTD